src/main/java/gov/cabinetofice/gapuserservice/dto/ChangeDepartmentDto.java

```diff
@@ -0,0 +1,19 @@
+package gov.cabinetofice.gapuserservice.dto;
+
+import gov.cabinetofice.gapuserservice.validation.annotations.FieldNotNull;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ChangeDepartmentDto {
+    @FieldNotNull(message = "Select a department")
```

Comment:
Maybe one for the new year rather than now, but can we not just use `@NotNull` from 'javax.validation.constraints.notnull` for this rather than defining our own annotation and validator? We do that elsewhere, eg in the admin backend:

<img width="457" alt="image" src="https://github.com/cabinetoffice/gap-user-service/assets/135321532/f8032eeb-71a6-4bcf-9a5b-0a5e8e2d08a1">

Reply:
Definitely worth giving a proper go after the new year, but there I tried it with the @NotNull annotation instead and it errors out, may be my local setup, although could be why Dylan advised creating a custom one (I can't quite remember). 

https://github.com/cabinetoffice/gap-user-service/assets/145372589/a8e988bb-8252-46f6-9498-ef733edfad72